src/main/kotlin/com/github/ashleybudtke/happyplants/domain/PlantServiceImpl.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.ashleybudtke.happyplants.domain
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class PlantServiceImpl : PlantService {
```

Comment:
Why have this implement `PlantService`? Not sure the value in having that interface unless there's something coming soon.

Reply:
Following SOLID principles out of habit, but I agree it's probably unnecessary for such a tiny project.